test/e2e/tests/wp-admin/settings.spec.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('Settings', () => {
+	test('A link to the plugin settings page is present under the Events menu', async ({
+		page,
+		admin,
+	}) => {
+		await admin.visitAdminPage('/');
+
+		const menu = page.locator('li', {
+			has: page.getByRole('link', { name: 'Events' }),
+		});
+		const settingsMenu = menu.getByRole('link', {
+			name: 'Settings',
+		});
+		const settingsMenuUrl = await settingsMenu.getAttribute('href');
+
+		await expect(settingsMenu).toBeVisible();
+		await expect(settingsMenuUrl).toContain(
+			'admin.php?page=gatherpress_general'
+		);
```

Comment:
The URL attribute should be asserted via the [`toHaveAttribute`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute) locator assertion so that it's properly awaited:

```suggestion
		const settingsMenu = menu.getByRole('link', {
			name: 'Settings',
		});

		await expect(settingsMenu).toBeVisible();
		await expect(settingsMenu).toHaveAttribute(
			'href',
			/admin\.php\?page=gatherpress_general/
		);
```

Reply:
Looks much better, thanks.